MdocDataTransfer.podspec

```diff
@@ -0,0 +1,35 @@
+require 'json'
```

Comment:
Why is this in the root of the project right now?

Reply:
that's how you do it with react native modules